sdk/streamlib-python-wheel/python/streamlib/audio_block.py

```
@@ -0,0 +1,184 @@
+            not _is_plain_int(sample_rate)
+            or not _is_plain_int(channels)
+            or not _is_plain_int(sample_count)
+            or not _is_plain_int(first_sample_timestamp_ns)
+        ):
+            raise ValueError(
+                "bag is not an audio block: sample_rate/channels/sample_count/"
+                "first_sample_timestamp_ns must be int"
+            )
+        # Before the length check, which two negatives would slip through by
+        # cancelling: a payload of four bytes satisfies
+        # `sample_count=-1 × channels=-1 × 4`, and the block would then fail at
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)